db/models/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const Sequelize = require('sequelize');
+const debug = require('debug')('tnlcf:models');
+const appData = require('../../common/appData');
+
+const env = process.env.NODE_ENV || 'development';
+const databaseConfig = require('../database-config')[env];
+
+const Feedback = require('./feedback');
+const SurveyAnswer = require('./survey');
+
+debug(`Using ${databaseConfig.dialect} database`);
+
+const sequelize = new Sequelize(databaseConfig.url, databaseConfig);
+
+sequelize
+    .authenticate()
+    .then(() => {
+        debug('Connection has been established successfully.');
+    })
+    .catch(err => {
+        debug('Unable to connect to the database:', err);
+    });
+
+/**
+ * Register models and associations
+ */
+const db = {
+    Users: sequelize.import('./user'),
+    Staff: sequelize.import('./staff'),
+    Feedback: Feedback.init(sequelize, Sequelize),
+    SurveyAnswer: SurveyAnswer.init(sequelize, Sequelize),
```

Comment:
When using classes for models you call `.init` on the model, rather than `sequelize.import`